frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/commentsPanel/comment/comment.component.tsx

```diff
@@ -43,7 +44,7 @@ export const Comment = ({
 }: CommentProps) => {
 	const [commentAge, setCommentAge] = useState('');
 
-	const isCurrentUser = CurrentUserHooksSelectors.selectUsername() === author;
+	const isCurrentUser = CurrentUserHooksSelectors.selectUsername() === (originalAuthor || author);
```

Comment:
I dont think this is necessary. originalAuthor doesnt correspond to any user in 3drepo

Reply:
I think this was in an old commit, this was removed in the last commits